R/Model-class.R

```diff
@@ -1650,7 +1650,7 @@ DualEndpointRW <- function(sigma2betaW,
 #' @details This class extends the [`DualEndpoint`] class so that the dose-biomarker
 #'   relationship \eqn{f(x)} is modelled by a parametric, rescaled beta density
 #'   function:
-#'   \deqn{f(x) = E0 + (Emax - E0) * Beta(delta1, delta2) * (x/x*)^delta1 * (1 - x/x*)^delta2,}
+#'   \deqn{f(x) = E0 + (Emax - E0) * Beta(delta1, delta2) * (x/x*)^delta1^ * (1 - x/x*)^delta2^,}
```

Comment:
not sure about this change, because the `^` was for exponentiating, not for highlighting or so

Reply:
This was the original documentation:
![image](https://github.com/Roche/crmPack/assets/133956382/8f5b3146-009d-44ed-b20d-9ff5118d1bcd)
I believe the exponents should be `delta1` and `delta2`, not just `d` with the `elta1` and `elta2` as some sort of "dangling multipliers"...

Reply:
ah I see, do we need to write then `(x/x*)^{\delta_1}` or is that too much LaTeX for R to handle?